### DISCUSSION TITLE
Obtain size and weight information of products measured in official warehouses

### DISCUSSION THREAD

**Discussion by yao657355700** [allegro/allegro-api#8828] [Suggestions - rejected]
Because the operating costs of the order are only in the form of a table, it is not possible to obtain the operating costs of the order through the API.

I need to calculate myself based on the size of the product, but there is no API interface to obtain information about the product size

**Comment by MaciejFrackowiak** [comment]
You have already posted such a suggestion in the thread https://github.com/allegro/allegro-api/discussions/8174.

**Reply by yao657355700** [reply]
Is there an API interface that can be implemented?

**Reply by MaciejFrackowiak** [reply]
Your suggestion is under verification.

**Reply by yao657355700** [reply]
I hope it can be resolved soon.
We have a lot of trouble calculating costs.
